guides/space_status_history.md

```diff
@@ -0,0 +1,56 @@
+---
+copyright: 'Copyright IBM Corp. 2017'
+link: 'space-status-history'
+is: 'experimental'
+---
+
+# Space Status History
+
+## Concepts
+
+The _statusHistory_ field provides a list containing information about status changes 
+in a space. The most recent value for each space status option is stored, ie. when a value is selected multiple times the old values are overwritten. 
+For a new space initially the statusHistory will be empty. 
+A statusHistory entry will be created when a user selects a new status value in the space.
+
+The statusHistory data is eventually consistent and not synchronized with a status update.
+
+## Schema
+
+### Space Status History Fetcher
+
+```grapqhl
+query  {
+  space(id: "space-id") {
+    statusHistory {
+      id
```

Comment:
@andrewegan 
What does this `id` refers to?  Is it used to correlate the actual change somewhere else?   If not, then the semantics used to name this may be misleading.  Perhaps it is not a `status history` but rather a status change log/history? (ie. time date values) of who changed the status regardless of what the status value was?

Also, some sample use case may help.... at this point one wonders why is would be important to know that Mary changed the status for space X on January 3rd if the previous or new status values are already lost due to subsequent status changes...  Hope this made sense :-)

Reply:
`id` here is the ID of the status that was set.

Reply:
> at this point one wonders why is would be important to know that Mary changed the status for space X on January 3rd if the previous or new status values are already lost due to subsequent status changes...

From a UI point of view it allows you to display the last setting for all possible status values. But it is limited and could be expanded to track more or all of the history in future.